### ISSUE TITLE
Issue on CreateTrigStmt "WHEN" condition

### ISSUE THREAD

**Issue by rdunklau** [lelit/pglast#18]
Hello, 

In the process of removing my overrides and falling back to the new release of pglast, I noticed a bug in the CreateTrigger statement: the "when" condition must always be wrapped in a set of parentheses, for example:

```
CREATE TRIGGER trig1
  AFTER UPDATE
  ON schema1.table1
  FOR EACH ROW
    WHEN (new.col1 IS DISTINCT FROM old.col1)
    EXECUTE PROCEDURE schema1.func1()
```

Is "prettified" as 
```
CREATE TRIGGER trig1
  AFTER UPDATE
  ON schema1.table1
  FOR EACH ROW
    WHEN new.col1 IS DISTINCT FROM old.col1
    EXECUTE PROCEDURE schema1.func1()
```
which is not valid syntax.

**Comment by lelit** [comment]
Thanks for the report!

**Comment by lelit** [comment]
This has been released as of version 1.4.
